script.skin.helper.service/default.py

```diff
@@ -99,36 +114,50 @@ def __init__(self):
                 if resultAction:
                     if "jsonrpc" in resultAction:
                         xbmc.executeJSONRPC(resultAction)
+                        xbmc.executeJSONRPC(resultAction)
```

Comment:
Does the response tell you if it was successful?. If yes, then you should probably check that first before triggering a second one.